packages/cortex-react/src/tests/Snackbar.test.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import { BaseSnackbar } from '../components';
+import { render, screen } from '@testing-library/react';
+
+describe('Snack', () => {
+  it('Should render Snack', () => {
+    render(
+      <BaseSnackbar show={true} variants={{ intent: 'default' }}>
+        <p>Description snackbar</p>
+      </BaseSnackbar>
+    );
+    const snack = screen.getByTestId('snackbar');
+    expect(snack).toBeInTheDocument();
+  });
+
+  it('Dont Should render Snack', () => {
```

Comment:
só pra manter o padrãozinho da descrição trocaria para "Should not ..."  ou "Shouldn't ..."

Reply:
ok